addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16814,7 +16814,7 @@ msgstr ""
 #. This string is an early meme from the late 1990's that made fun of the poor translation in the 16-bit game Zero Wing from the late 1980's. DO NOT TRANSLATE!
 #: system/settings/settings.xml
 msgctxt "#35200"
-msgid "ALL YOUR BASE ARE BELONG[CR]TO US"
+msgid "ALL YOUR BASE ARE BELONG TO US"
```

Comment:
Can the [CR] here remain? Part of the history of this joke involves the incongruous line break